packages/bff/src/services/catalogService.ts

```diff
@@ -129,6 +124,36 @@ const getBulkAttributes = async (
   return await attributesBulk(0, []);
 };
 
+const getEserviceDesciptor = (
```

Comment:
```suggestion
const retrieveEserviceDesciptor = (
```
We can align this name with the other retrieves like this

Reply:
Done with this [commit](https://github.com/pagopa/interop-be-monorepo/pull/712/commits/7dc73a20f75f65a16e938f00100ea8da9b22919f#diff-7efff3cb13375394a1259ed6f3d5989f9280230ee579598a3d471ce60fd811aa)